pkg/api/folder.go

```diff
@@ -226,7 +222,7 @@ func (hs *HTTPServer) setDefaultFolderPermissions(ctx context.Context, orgID int
 
 	var permissions []accesscontrol.SetResourcePermissionCommand
 
-	if user.IsIdentityType(claims.TypeUser) {
+	if user.IsIdentityType(claims.TypeUser, claims.TypeServiceAccount) {
```

Comment:
We also need to set permissions for service accounts, looks like this got lost through some refactors: https://github.com/grafana/grafana/pull/95072